src/components/CoralIcon.jsx

```diff
@@ -0,0 +1,76 @@
+import React, {PropTypes} from 'react';
+import Radium from 'radium';
+import _ from 'lodash';
+
+@Radium
+class CoralIcon extends React.Component {
+
+  static propTypes = {
+    name: PropTypes.string.isRequired
+  }
+
+  getColorProps(color, isInverse) {
+
+    if (isInverse) {
+      return { backgroundColor: color, color: 'white' };
+    } else {
+      return { color: color };
+    }
+  }
+
+  render() {
+    var g = _.find(icons, {name: this.props.name}).glyph || icons[0].glyph;;
+    return (
+      <div style={[
+        this.props.style,
+        styles.base,
+        this.props.size === 'small' && styles.small,
```

Comment:
you can replace this 3 lines with one: `styles[this.props.size]`. If this don't match any style it will be undefined and not apply